tests/battery_mod_test.cpp

```diff
@@ -342,6 +342,7 @@ TEST_CASE( "installing battery in tool", "[battery][tool][install]" )
 
         // Should fail to install the magazine
         REQUIRE( flashlight.contents.has_pocket_type( item_pocket::pocket_type::MAGAZINE_WELL ) );
+        // TODO capture debugmsg PR #41677 will be merged
```

Comment:
It's not clear what these TODO comments mean; can you expand them a little.

Reply:
Ah, I think I understand after re-reading your PR description.  Maybe change this to
```
// TODO capture and vefify debugmsg output once PR #41677 has been merged
```

Reply:
I have already added another comment. In fact, this is TODO for me. I will do it either in this PR or PR #41677, whichever PR is the second. It was also added to trigger a merge conflict so I will not forget to resolve it.